easybuild/easyconfigs/i/Infomap/Infomap-20190308-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,27 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'MakeCp'
+
+name = 'Infomap'
+version = '20190308'
+local_commit = 'bd65c71'
+
+homepage = 'https://www.mapequation.org/code.html#Linux'
+description = """Multi-level network clustering based on the Map equation."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+toolchainopts = {'pic': True}
+
+files_to_copy = ['%(name)s']
```

Comment:
@omula style nitpicking: please move this down below `checksums`, with an empty line in between